helm/management-cluster-admission/templates/kustomize-out.yaml

```diff
@@ -258,9 +258,9 @@ data:
     apiVersion: controller-runtime.sigs.k8s.io/v1alpha1
     kind: ControllerManagerConfig
     health:
-      healthProbeBindAddress: :8081
+      healthProbeBindAddress: '{{ .Values.healthProbeBindAddress }}'
     metrics:
-      bindAddress: 127.0.0.1:8080
+      bindAddress: '127.0.0.1:{{ .Values.metricsBindAddress }}'
```

Comment:
In general LGTM (even though @rossf7 said it won't help much) but I'm not sure why this is changed